.github/actions/release-candidate-ready-for-qa-v1/action.yml

```diff
@@ -0,0 +1,87 @@
+name: release-candidate-ready-for-qa-v1
+description: Preparing a release candidate issue for QA
+
+inputs:
+  sha-rc:
+    description: SHA of the release commit from the release branch
+    required: true
+  slack-webhook:
+    description: A Slack channel webhook URL where the message will be sent
+    required: true
+  owner:
+    description: 'The owner of the repository e.g. dequelabs/axe-core-npm, then supply "dequelabs"'
+    required: false
+  repo:
+    description: 'The repository e.g. dequelabs/axe-core-npm, then supply "axe-core-npm"'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Get owner and repository name
+      id: get-owner-repo-name
+      shell: bash
+      run: |
+        if [[ -n "${{ inputs.owner }}" ]]; then
+          ORGANIZATION="${{ inputs.owner }}"
+        else
+          IFS='/' read -r ORGANIZATION _ <<< "$GITHUB_REPOSITORY"
+        fi
+
+        if [[ -n "${{ inputs.repo }}" ]]; then
+          REPOSITORY="${{ inputs.repo }}"
+        else
+          IFS='/' read -r _ REPOSITORY <<< "$GITHUB_REPOSITORY"
+        fi
+
+        echo "owner=$ORGANIZATION" >> $GITHUB_OUTPUT
+        echo "repo=$REPOSITORY" >> $GITHUB_OUTPUT
```

Comment:
You don't need to do this. `repo` and `owner` have default values in https://github.com/dequelabs/axe-api-team-public/blob/main/.github/actions/get-release-issue-v1/src/run.ts#L8-L9 so you can just forward the `repo` and `owner` inputs directly and let the release issue action use the defaults if none are provided.

Reply:
In the [get-release-issue-v1](https://github.com/dequelabs/axe-api-team-public/blob/main/.github/actions/get-release-issue-v1/src/run.ts#L8-L9) these are optional params so I leave it as in that action. Also, the repo name is used for adding a comment to an issue https://github.com/dequelabs/axe-api-team-public/blob/ci-add-deploy-release-candidate/.github/actions/release-candidate-ready-for-qa-v1/action.yml#L63

Reply:
also, I will remove `inputs.owner`,` inputs.repo` because they are not needed